common/presenters/table/object-to-table-head.js

```diff
@@ -1,17 +1,49 @@
 const { pickBy } = require('lodash')
+const queryString = require('query-string')
 
 const i18n = require('../../../config/i18n')
 
-function objectToTableHead({ head } = {}) {
-  if (!head) {
-    return undefined
+function buildUrl(head) {
+  const { query, key } = head
+  return queryString.stringify({
+    ...query,
+    sortBy: key,
+    sortDirection: (query && query.sortDirection) === 'asc' ? 'desc' : 'asc',
```

Comment:
Using a default for `query` in the arguments destructuring would allow you to drop the check for `query` here and simplify this line a little:

```javascript
sortDirection: query.sortDirection === 'asc' ? 'desc' : 'asc',
```